src/textual/css/_styles_builder.py

```diff
@@ -441,6 +441,7 @@ def _process_space_partial(self, name: str, tokens: list[Token]) -> None:
     def _parse_border(self, name: str, tokens: list[Token]) -> BorderValue:
         border_type: EdgeType = "solid"
         border_color = Color(0, 255, 0)
+        border_alpha: float | None = None
```

Comment:
Is the None here instead of 100.0 to differentiate between someone who has explicitly set 100% and someone who hasn't gave a percentage?

Would it make sense to default it to 100.0?

Have you checked the interactions when we default to None here and a user tries to animate the border percentage to e.g. 0% using code?

Reply:
> Is the None here instead of 100.0 to differentiate between someone who has explicitly set 100% and someone who hasn't gave a percentage?
>
> Would it make sense to default it to 100.0?

Aye, it started out that way (following the example set by the `color` style) but now you mention it it can likely be reduced to that.

Reply:
Actually, thinking about it some more, I think it's better as it is in that if we go with `None` it saves the need to create a fresh instance of a `Color` just to end up with the same `Color`. Would save a wee bit of time I think?

Reply:
> vHave you checked the interactions when we default to None here and a user tries to animate the border percentage to e.g. 0% using code?

This part I don't think I follow.

Reply:
The part about animation is less relevant if this can't be set inline.